frontend/src/libs/components/button/button.tsx

```diff
@@ -1,11 +1,36 @@
+import { IconName } from '~/libs/enums/icon-name.enum.js';
+import { getValidClassNames } from '~/libs/helpers/helpers.js';
+
+import { Icon } from '../components.js';
+import styles from './styles.module.scss';
+
 type Properties = {
   label: string;
   type?: 'button' | 'submit';
+  size?: 'sm' | 'md';
+  variant?: 'contained' | 'outlined' | 'text';
+  isDisabled?: boolean;
+  onClick?: () => void;
 };
 
 const Button: React.FC<Properties> = ({
   type = 'button',
+  size = 'md',
+  variant = 'contained',
   label,
-}: Properties) => <button type={type}>{label}</button>;
+  isDisabled = false,
+  onClick,
+}: Properties) => (
+  <button
+    className={getValidClassNames(styles.btn, styles[size], styles[variant])}
+    type={type}
+    disabled={isDisabled}
+    onClick={onClick}
+  >
+    <Icon iconName={IconName.PLUS} />
```

Comment:
I do not think that we should always render the PLUS and CHEVRON_DOWN icons. 
Let's consider expanding the label configuration to support the next abilities 
1) Optional icon before label 
2) Optional icon after label  

The easiest way is to make a label string or react element, to allow the dev to pass whatever he wants to the button. 
If there are any other ideas you are welcome 

P.S other than that everything seems OK

Reply:
Yes, that's right, it's better to give the ability to pass different icons.